Client/Shared/RightMenu/RightStandard.razor

```diff
@@ -0,0 +1,54 @@
+﻿@inject IUsersRepository _usersRepository
+<div class="box">
+    <h4>About</h4>
+    <p>
+        Do you ever find yourself in need of some information but don't know where to find it?
+        Quanda is the answer.
+        Quanda is a questions and answers page for information exchange.
+        It allows you to ask questions and other to answer it.
+        Main purpose of page is to share knowlege.
+        It's a place where you can share your questions and get answers.
+    </p>
+</div>
+<div class="box">
+    <h4>Top 3 users</h4>
+    @if (Top3Users != null)
+    {
+        @foreach (var user in Top3Users.Select((value, index) => new { index, value }))
```

Comment:
Why do you use Select in this case?

Reply:
Because I need index to generate css class in line 20. This index describes if trophy is gold, silver or bronze.
Default foreach has no index, so i needed to created custom one.

Reply:
Ok, I got it.